packages/reg-suit-cli/src/commands/compare.ts

```diff
@@ -4,7 +4,13 @@ import getRegCore from "../get-reg-core";
 async function compare(options: CliOptions) {
   const processor = getRegCore(options).createProcessor();
   const ctx = await processor.getExpectedKey();
-  await processor.compare(ctx);
+  const { comparisonResult } = await processor.compare(ctx);
+  const numberOfFailed = comparisonResult.failedItems.length;
```

Comment:
we may also want to fail if there are new items or deleted, but I did not do that for now